.github/workflows/release-wc-beta-tester.yml

```diff
@@ -0,0 +1,33 @@
+name: WooCommerce Beta Tester Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: 'The version number for the release'
+                required: true
+
+jobs:
+    release:
+        name: Run release scripts
+        runs-on: ubuntu-20.04
+        permissions:
+            contents: write
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup WooCommerce Monorepo
+              uses: ./.github/actions/setup-woocommerce-monorepo
+
+            - name: Build WooCommerce Beta Tester Zip
+              working-directory: plugins/woocommerce-beta-tester
+              run: pnpm build:zip
+
+            - name: Create release
+              id: create_release
+              uses: samueljseay/action-gh-release@master
```

Comment:
I ended up forking the original action to add support for marking the release as "not latest". My plan is to upstream a PR for this fix to https://github.com/softprops/action-gh-release